apps/console-electron/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@dicedb/console-electron",
+  "version": "0.1.0",
+  "private": true,
+  "main": "main.js",
+  "type": "module",
+  "scripts": {
+    "dev": "concurrently --kill-others \"next dev -p ${PORT-4000}\" \"electron .\"",
```

Comment:
Shouldnt we be running next from inside electron? How will this be packaged when we run the built version?

Reply:
This is how I am thinking, let me know if I am missing something
- `local` We start nextjs server and our electron points to the localhost URL.  This way we get nice HMR.
- `packaged` We build nextjs, and then point electron to use generated build assets in packaged version.  We won't have a separate nextjs server running.  It also means that probably we'll bundle all the generated assets with the electron packaged version.

Reply:
It makes sense to have standalone nextjs server running and electron app is pointing to it, for both `local` and `packaged` versions.  There is no significant advantage from just shipping bundled files, not the server in packaged version.  

Just one issue - users can run this directly also outside electron if they know the URL which shouldn't be an issue.

Reply:
I would prefer the server runs on the users system, and not us hosting it. Especially because we’ll actual servers if we wanna support load tests

Reply:
If you mean that the server for nextjs is gonna run in packaged version on the users system, what invokes it in the package version?

there is nothing calling next start

Reply:
Similar to forking a nodejs server, we can call `childProcess.spawn('pnpm', ['start'])` or something like that.

Reply:
Not sure that bundling pnpm in the final export is that good of an idea.

id rather just call node server.js on the bundled server output

Reply:
@KaviiSuri That was top of my head. Definitely we won't bundle pnpm in final build 😆